src/Playwright/Transport/Channels/ArtifactChannel.cs

```diff
@@ -46,9 +47,9 @@ internal async Task SaveAsAsync(string path)
         => await Connection.SendMessageToServerAsync<JsonElement>(
             Guid,
             "saveAs",
-            new
+            new Dictionary<string, object>
```

Comment:
This looks like a `Dictionary<string, string>`

Reply:
ditto